lib/etcd_databag.py

```diff
@@ -94,3 +98,16 @@ def storage_path(self):
             return "/media/etcd/data"
         else:
             return etcd_opts['etcd_data_dir']
+
+    def get_bind_address(self):
+        ''' Returns the address that the service binds to. The value is taken
+        from the config parameter 'service_network_interfaces'.
+        If value has been set to 'public' or 'private', the unit public or
+        private address is considered. Otherwise, service is bound to any
+        available network interface'''
+        service_network_iface = config('service_network_interface')
+        if 'private' in service_network_iface:
```

Comment:
This will be true if I set it to 'totallybrokenbuthasprivateinsideit'. I would think we would just want to compare the string to private and public.